src/interfaces/IECDSAStakeRegistry.sol

```diff
@@ -252,6 +254,26 @@ interface IECDSAStakeRegistry is
         address operator
     ) external view returns (bool);
 
+    /*
```

Comment:
Can you reorganize these to be next to the similar methods like `getLatestOperatorSigningKey`?

The names are a bit confusing but I cannot think of better ones. I kind of feel the orignal one should be `getLatestSigningKeyForOperator` but we can't change the interface here without breaking stuff. And calling this one `getLatestSigningKeyOperator` would be very odd.

Naming is hard, but we can leave it like this.